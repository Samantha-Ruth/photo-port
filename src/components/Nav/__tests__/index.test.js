import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Nav from '..';

afterEach(cleanup);
describe('Nav component', () => {
    // First Test
     it('renders', () => {
        render(<Nav />);
      });

    // Second Test
    it('matches snapshot DOM node structure', () => {
        // render Nav
        const { asFragment } = render(<Nav />);
        expect(asFragment()).toMatchSnapshot();
      });
})