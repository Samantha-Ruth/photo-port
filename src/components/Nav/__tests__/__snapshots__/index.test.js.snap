// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <header
    class="flex-row px-1"
    data-testid="header"
  >
    <h2>
      <a
        href="/"
      >
        <span
          aria-label="camera"
          role="img"
        >
           ðŸ“¸
        </span>
         Oh Snap!
      </a>
    </h2>
    <nav>
      <ul
        class="flex-row"
      >
        <li
          class="mx-2"
        >
          <a
            href="#about"
          >
            About me
          </a>
        </li>
        <li
          class="mx-2"
        >
          <span>
            Contact
          </span>
        </li>
        <li
          class="mx-1"
        >
          <span>
            Commercial
          </span>
        </li>
        <li
          class="mx-1"
        >
          <span>
            Portraits
          </span>
        </li>
        <li
          class="mx-1"
        >
          <span>
            Food
          </span>
        </li>
        <li
          class="mx-1"
        >
          <span>
            Landscape
          </span>
        </li>
      </ul>
    </nav>
  </header>
</DocumentFragment>
`;
